### ISSUE TITLE
AI: Detect "Drawings" and "Paintings"

### ISSUE THREAD

**Issue by ant0nwax** [photoprism/photoprism#3243]
I currently search for two additional labels that are missing for my daily use:
Drawings and Paintings, i used them a lot on my former photo cloud and the AI there worked better.

This FR is only for adding the particular Labels Drawings and Paintings by the use of AI, can this be done more easily? I guess some deep learning on these two topics would have to be done, and here we go.

(Also the OCR feature of text stored on Photos is a thing that could be a nice to have integration by the developers, but for that there is a separate thread )

**Comment by lastzero** [comment]
Maybe this is possible already. Could you provide us with a number of pictures you expect to be classified as painting?

http://wordnetweb.princeton.edu/perl/webwn?s=painting

**Comment by ant0nwax** [comment]
@lastzero could you please explain a little bit as a human, (your message seems to be auto-generated phising for princton.edu) before i share data to you, the FR is not one category called Paintings the issue is Paintings as one Label and Drawings as another label...

I help you for getting content for deep learning you do not need my photos, they are only few but the following links help you better

PAINTINGS
https://search.brave.com/images?q=paintings+styles&source=web

DRAWINGS
https://search.brave.com/images?q=drawings+drafts&source=web


just crawl them and downlaod all of them and voila you have the content for the deep learning

**Comment by lastzero** [comment]
I'm the maintainer of this project and my messages are never auto generated. We'll take a look at this as soon as possible, but note we are a small team and the roadmap is packed already.

**Comment by ant0nwax** [comment]
thank you @lastzero for the heads up... I would really appreciate it and buy you or your team a coffee during you started the PR

**Comment by ant0nwax** [comment]
Dear photoprism github community,

did anyone find a solution how to find all drawings or paintings in their huge stack of photos?

**Comment by graciousgrey** [comment]
You can now use our Ollama or OpenAI integrations to generate labels and captions.

- https://docs.photoprism.app/user-guide/ai/using-ollama/
- https://docs.photoprism.app/user-guide/ai/

I am using the QWEN3VL model. For paintings and drawings in my library, it creates labels such as "painting," "drawing," or "watercolor."

**Comment by ant0nwax** [comment]
Thank you for this answer, you seem to be a real amazone :)
Unluckily at the moment in my country the .app domain is blocked by the great wall, i know workarounds more modern than proxies, ask AI in order to read URLs for me.
